CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+## 0.0.7
```

Comment:
Let's hold off on bumping the version just yet, there are a couple PRs in flight. I'll cut a new release by EOD regardless of where those are.